js/src/analysis/handleWorkerError.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { __ } from "@wordpress/i18n";
+import interpolateComponents from "interpolate-components";
+import { setWarningMessage } from "../redux/actions/warning";
+import getL10nObject from "./getL10nObject";
+
+/**
+ * Dispatches a warning message if the recalibration analysis fails.
+ *
+ * @returns {void}.
+ */
+export default function handleWorkerError() {
+	const isRecalibrationBetaActive = getL10nObject().recalibrationBetaActive;
+
+	let message = [];
+	if ( isRecalibrationBetaActive ) {
+		// Disable reason: the anchor has content, as the component uses interpolateComponents.
+		/* eslint-disable jsx-a11y/anchor-has-content */
+		message = interpolateComponents( {
+			/* Translators:
+			 * {{link}} resolves to the link to SEO/Features,
+			 * {{/link}} resolves to the link closing tag,
+			 * {{bold}} resolves to a bold opening tag (<b>),
+			 * {{/bold}} resolves to a bold closing tag.
+			 */
+			mixedString: __(
+				"We're sorry! Unfortunately, the recalibrated analysis beta doesn't work as intended with your current " +
+				"setup {{bold}}yet{{/bold}}. " +
+				"Please {{link}}deactivate the recalibration beta in your features{{/link}} and please try again later. " +
+				"We value your input!",
+				"wordpress-seo"
+			),
+			components: {
+				bold: <b />,
+				link: <a href="/wp-admin/admin.php?page=wpseo_dashboard#top#features" target="_blank" rel="noopener noreferrer" />,
```

Comment:
This will not work on all installations - subdirectory installs or installs where the admin has a different URL.
Please use https://codex.wordpress.org/Function_Reference/admin_url to determine the full URL.

Reply:
This triggered me, users do not necessarily have access to alter this setting. @manuelaugustin and I thought it would now be better to just remove the url. Instead of the trickier testing for the user permissions and having multiple messages. Because the use-case is very small, we don't think it is necessary to implement all this functionality.

The updated message:
We're sorry! Unfortunately, the recalibrated analysis beta doesn't work as intended with your current setup **yet**. Please deactivate the recalibration beta under "SEO - General - Features" and please try again later. We value your input! If you can't access the feature page, please contact your administrator.